client.go

```
@@ -41,71 +41,97 @@ import (
+	// GetChangeListeners returns the list of configuration change listeners
 	GetChangeListeners() *list.List
+	// UseEventDispatch enables event dispatch for configuration changes
 	UseEventDispatch()
+	// Close stops the configuration polling
 	Close()
 }
 
-// internalClient apollo 客户端实例
+// internalClient represents the internal implementation of the Apollo client
 type internalClient struct {
 	initAppConfigFunc func() (*config.AppConfig, error)
```

golangci-lint (1.64.8)

field `initAppConfigFunc` is unused

(unused)